### PR TITLE
add p4sdk container image build steps

### PR DESCRIPTION
**Pull request by ahalimx86**
Adding P4 SDK container image build steps

### CODE REVIEW

File: p4sdk/images/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Intel Corporation
+FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as builder
```
Comment (Eoghan1232) [inline p4sdk/images/Dockerfile:3]: update Hadolint ci job to ignore for this DL3029 warning as the --platform is required

File: p4sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+## Initialize Emulation Support
+
+```
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes -c yes
+
+```
+
+### Extract IPU SDK source code
+```
+cp zxvf intel-ipu-sdk-source-code-1.2.0.7550.tar.gz /tmp
```
Comment (Eoghan1232) [inline p4sdk/README.md:19]:
```suggestion
cp intel-ipu-sdk-source-code-1.2.0.7550.tar.gz /tmp
```

File: p4sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+cd /tmp
+tar zxvf intel-ipu-sdk-source-code-1.2.0.7550.tar.gz
```
Comment (Eoghan1232) [inline p4sdk/README.md:21]:
```suggestion
tar -zxvf intel-ipu-sdk-source-code-1.2.0.7550.tar.gz
```

File: p4sdk/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+### Build ACC workbench
+This can take couple of hours to build.
+
+```
+nohup ./scripts/workbench/build_workbench.sh -t acc > acc_workbench.log 2>&1 &
+```
+You can watch the logs from:
+```
+tail -f acc_workbench.log
```
Comment (Eoghan1232) [inline p4sdk/README.md:33]: this is from current dir ?
i.e. Intel_IPU_SDK-7550

File: p4sdk/images/Dockerfile (hunk 1 above)
Comment (Eoghan1232) [reply p4sdk/images/Dockerfile:3]: this is now resolved in main

File: p4sdk/README.md (hunk 3 above)
Comment (ahalimx86) [reply p4sdk/README.md:21]: '-' is not needed :)

**Pull request merged**